argus/action_manager/windows.py

```diff
@@ -166,11 +166,12 @@ def git_clone(self, repo_url, location):
             Specific location on the instance.
         """
         LOG.info("Cloning from %s to %s", repo_url, location)
-        cmd = "git clone {} {}".format(repo_url, location)
+        cmd = ("If (!(Test-Path -path {})){ git clone {} {} }".
```

Comment:
If something happens during the `git clone` that issue will be ignored and argus will use a incomplete / invalid copy of the repository. A better approach will be to remove the directory if something happens and let argus try again.